convert2rhel/data/7/x86_64/configs/centos-7-x86_64.cfg

```
@@ -5,6 +5,10 @@
 # Delimited by whitespace(s).
 gpg_fingerprints = 24c6a8a7f4a80eb5
 
+# Release package used to query the major and minor version
+release_pkg =
+  centos-release
```

I don't think we will have more than one package here at time, so, what do you think about inlining this package to not confuse people into thinking that this may be a list? 


```suggestion
release_pkg = centos-release
```

Its a minor suggestion, but inlining here could help thinking that this is treated as a single item